### DISCUSSION TITLE
Live previews with nnn in foot terminal?

### DISCUSSION THREAD

**Discussion by b4shful** [jarun/nnn#1959] [General]
Do any of the current plugins for live preview work with the [foot terminal](https://codeberg.org/dnkl/foot)?

If anyone has any config tips for using live previews and foot, would be cool.

**Comment by N-R-K** [comment]
Use tmux + img2sixel with `preview-tui`. It should work since foot supports sixel. `preview-tui` file has a lot of instruction at the top. There's also some instructions [at the wiki](https://github.com/jarun/nnn/wiki/Live-previews).
